docs/specs/modules/booking.definitions.yaml

```diff
@@ -63,15 +63,15 @@ Purchase:
       type: string
     id:
       type: string
-    validFrom:
+    validFromTimestamp:
       type: number
-#        description: in seconds since 1970
-    timezone:
-      type: string
-#        desription: timezone ID for validFrom
+      description: datetime of when the booking starts in ISO 8601 (e.g., `2017-05-30T12:00:00+10:00`)
     validFor:
       type: number
-#        description: duration in seconds
+      description: duration in seconds
+    pickupWindowDuration:
+      type: number
+      description: duration in seconds,
```

Comment:
Can you add to the description how this be interpreted? It's not clear from the name.

Reply:
@MarianoTucat Reminder ^^